{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-14330",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-14330",
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-14330",
                    "refsource": "CONFIRM",
                    "tags": []
                },
                {
                    "url": "https://github.com/ansible/ansible/issues/68400",
                    "name": "https://github.com/ansible/ansible/issues/68400",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "An Improper Output Neutralization for Logs flaw was found in Ansible when using the uri module, where sensitive data is exposed to content and json output. This flaw allows an attacker to access the logs or outputs of performed tasks to read keys used in playbooks from other users within the uri module. The highest threat from this vulnerability is to data confidentiality."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-09-11T18:15Z",
    "lastModifiedDate": "2020-09-11T18:15Z"
}